keywords.robot

```diff
@@ -2978,3 +2978,70 @@ Should Contain All
     FOR    ${substring}    IN    @{substrings}
         Should Contain    ${string}    ${substring}
     END
+
```

Comment:
Please conform to: https://github.com/Dasharo/open-source-firmware-validation#guidelines before moving from draft